src/objects/Mesh.js

```diff
@@ -49,6 +49,59 @@ class Mesh extends Object3D {
 
 	}
 
+	volume( precision ) {
```

Comment:
I think its not appropiate since that would be the scene's volume an not the mesh's volume; I would add a scene method to get all individual volumes added. 

Not sure how often it could be used though

Reply:
We can also do a `MeshUtiks`...